subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -267,3 +267,7 @@ configurations {
 }
 ```
 
+[[war_convention_deprecation]]
+=== Deprecated `war` plugin conventions
+
+link:{javadocPath}/org/gradle/api/plugins/WarPluginConvention.html[WarPluginConvention] is now deprecated and scheduled for removal in Gradle 8.0. Clients should configure the `war` task  directly. Also, link:{javadocPath}/org/gradle/api/DomainObjectCollection.html#withType-java.lang.Class-[tasks.withType(War.class).configureEach(...)] can be used to configure each tasks of type `War`.
```

Comment:
`... can be used to configure each tasks ...` -> `... can be used to configure each task ...`

Reply:
Fixed.